packages/reakit/src/Toolbar/__examples__/ToolbarWithPopover/index.tsx

```diff
@@ -0,0 +1,43 @@
+import * as React from "react";
+import { useToolbarState, Toolbar, ToolbarItem } from "reakit/Toolbar";
+import {
+  usePopoverState,
+  Popover,
+  PopoverDisclosure,
+  PopoverArrow,
+  PopoverProps,
+} from "reakit/Popover";
+import { Button } from "reakit/Button";
+
+const PopoverItem = React.forwardRef<HTMLButtonElement, PopoverProps>(
```

Comment:
It should be `PopoverDisclosureProps` since `props` will be spread into `PopoverDisclosure`.
```suggestion
const PopoverItem = React.forwardRef<HTMLButtonElement, PopoverDisclosureProps>(
```